app/gui/src/project-view/stores/visualization/index.ts

```diff
@@ -248,13 +250,13 @@ export const [provideVisualizationStore, useVisualizationStore] = createContextS
       }
     })
 
-    function* types(type: Opt<string>) {
+    function* byType(type: Opt<ProjectPath>): IterableIterator<VisualizationIdentifier> {
       const types =
         type == null ?
           metadata.keys()
         : new Set([
-            ...(metadata.visualizationIdToType.reverseLookup(type) ?? []),
-            ...(metadata.visualizationIdToType.reverseLookup('Any') ?? []),
+            ...(metadata.visualizationIdToType.reverseLookup(type.toString()) ?? []),
```

Comment:
- `toString` may collide. `.key` is the serialization to use for maps.
- It seems we only use this index for "reverse" lookups, maybe we should replace it with a `visualizationIdByType`.